oshi-core/src/main/java/oshi/hardware/GlobalMemory.java

```diff
@@ -36,14 +36,14 @@
      *
      * @return Total number of bytes.
      */
-    long getTotal();
+    long getMemTotal();

```

Comment:
Whether it's an improvement for readability is also questionable. The class is already called GlobalMemory, so it's pretty obvious that a method called getTotal() would be getting the total memory.  And even then, it isn't saying it's getting the total memory, it's saying it's getting the total "mem". Whatever "mem" is.